architecture/core_concepts/routes.adoc

```diff
@@ -39,15 +46,56 @@ Configuration] guide for information on deploying a router.
+the service. Service-provided load balancing is bypassed and replaced with the
+router's own load balancing. Routers watch the cluster API and automatically
+update their own configuration according to any relevant changes to the IP
+addresses of pods that provide the service. Custom routers can be deployed to
+communicate modifications of API objects to an external routing solution.
 
-Router plug-ins assume they can bind to host ports 80 and 443. This is to allow
-external traffic to route to the host and subsequently through the router.
-Routers also assume that networking is configured such that it can access all
-pods in the cluster.
+In order to reach a router in the first place, requests for host names must
+resolve via DNS to a router or set of routers. The suggested method is to define
+a cloud domain with a wildcard DNS entry pointing to one or more virtual IP (VIP)
+addresses backed by multiple router instances on designated nodes.
+
+A single VIP along with multiple routers means that there will be just one
+active router and the other router(s) are hot-standby(s) - aka passive.
+With > 1 VIPs, more than one router could be in the mix and so one could have
+an "active-active" config (or even more complex setups ala "active-active-passive").
+
+Routes can be
+link:../../architecture/core_concepts/routes.html#router-sharding[sharded]
+among the set of routers. Administrators can set up sharding on a cluster
+wide basis and users can set up sharding for the namespace in their project.
+Sharding allows the operator to define multiple router groups. Each router
+in the group serves only a subset of traffic.
+
+Router VIP configuration is described in
+ifdef::openshift-enterprise,openshift-origin[]
+link:../../admin_guide/high_availability.html#configuring-a-highly-available-routing-service[High Availability].
+endif::[]
+ifdef::openshift-dedicated[]
+the https://docs.openshift.com/enterprise/3.1/admin_guide/high_availability.html#configuring-a-highly-available-routing-service[{product-title} Enterprise Cluster Administration documentation].
+endif::[]
+Routes using names and addresses outside the cloud domain require
+iconfiguration of individual DNS entries.
+
+
+If routes use host names/addresses outside the cloud domain, you will
+need to configure the DNS entries individually.
+
+{product-title} routers provide external host name mapping and load balancing
+of link:pods_and_services.html#services[service] end points over protocols that
+pass distinguishing information directly to the router; the host name
+must be present in the protocol in order for the router to determine
+where to send it.
+
+Router plug-ins assume they can bind to host ports 80 and 443, by default.  This
+allows external traffic to route to the host and subsequently through the router
+to a pod in the cluster.  Because a router binds to ports on the host node, only
+one router listening on those ports can be on each node. A router can listen on
```

Comment:
So there's a caveat here - routers that run with `--host-network=false` (not host networking aka container networking mode) don't have this restriction of one listener per port as they use the container's network. 
So maybe something like: 
```  
only one router listening on those ports can be on each node if the router uses host networking  
(`--host-network=true`).   
```